packages/cache/src/options.ts

```diff
@@ -112,10 +112,21 @@ export function getDownloadOptions(copy?: DownloadOptions): DownloadOptions {
       result.segmentTimeoutInMs = copy.segmentTimeoutInMs
     }
   }
-
+  const customDownloadTimeoutMins = process.env['CACHE_DOWNLOAD_TIMEOUT_MINS']
```

Comment:
Since the users are not aware of the internal working of the download client, hence kept it a generic one that everyone would understand. If we want to keep it `SEGMENT_DOWNLOAD_TIMEOUT_MINS`, we'll have to also explain more about what a segment is, how the download segment size differs in 32 bit and 64 bit systems, etc.

Reply:
That's a good point. It's just that customer might come back complaining that I set a timeout and the download timed out only after a longer time period. 
If it is helpful, we can document the process to download in segments and mention the timeout there. But agree that this is additional complexity.

Reply:
Will work on the documenting part then...

Reply:
@aparna-ravindra @anuragc617 any thoughts here?

Reply:
``CACHE_DOWNLOAD_TIMEOUT_MINS`` This is not complete cache download timeout, It is timeout for 2GB Should we rename to reflect this ? like CACHE_DOWNLOAD_TIMEOUT_PER_GB and we can compute the timeout for 2gb internally.

Reply:
Does [readme](https://github.com/actions/toolkit/blob/4b6b45fe18c542cabfb32eb24f78a67377aa5255/packages/cache/README.md#cache-segment-restore-timeout) work for the same? @tiwarishub